app/agent/activities/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from "@temporalio/activity";
+
+export const withActivityHeartbeat = <T>(
+  options: {
+    /** Default: 5000 */
+    intervalMs?: number;
+  },
+  fn: (...args: any[]) => Promise<T>,
+): Promise<T> => {
+  return new Promise((resolve, reject) => {
+    const interval = setInterval(() => {
+      Context.current().heartbeat();
```

Comment:
The default timeout is 10 seconds, where is that timeout set?

Reply:
In the `proxyActivities` function beside a workflow definition. For example here: https://github.com/hocus-dev/hocus/blob/8bcf3e17cc6bdc20179eb217f42e80b3342c99f4/app/agent/workflows/prebuild.ts#L28